FirebaseAppDistribution.podspec

```diff
@@ -43,7 +43,10 @@ iOS SDK for App Distribution for Firebase.
 
   s.test_spec 'unit' do |unit_tests|
    unit_tests.scheme = { :code_coverage => true }
-   unit_tests.source_files = 'FirebaseAppDistribution/Tests/Unit*/*.[mh]'
+   unit_tests.source_files = [
+     'FirebaseAppDistribution/Tests/Unit*/*.[mh]',
+     'FirebaseAppDistribution/Tests/Unit/Swift*/*.swift',
```

Comment:
Don't forget to `git add` the new tests.

Reply:
Whoops! Thanks